LIPS/lip-13.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Easy Track payments limit 
+tags: LIPs
+blockchain: ethereum
+lip: lip-13
+status: Implemented
+author: Alexey Potapkin, Eugene Pshenichnyy, Victor Suzdalev, Eugene Mamin
+discussions-to: https://research.lido.fi/t/lip-13-easy-track-payments-limit/1670
+---
+
+# Easy Track payment limits
+
+## Simple summary
+
+A simple security rule that limits Easy Track Motions while moving treasury funds.
+
+## Abstract 
+
+We propose adding the Aragon ACL permission that will limit an asset type and its amount for each payment initiated by Easy Track. The permission constrained with the limits will be set by DAO vote.
+
+## Motivation
+
+[Easy Track](https://github.com/lidofinance/easy-track) proved to be a safe and effective tool that helped streamline DAO operations and get rid of some routine votings. We want to extend it further, adding more Motion types, but some security concerns exist.
+
+Currently, Motions have access to the Lido treasury (the [DAO Agent](https://etherscan.io/address/0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c) contract) through [Finance](https://etherscan.io/address/0xB9E5CBB9CA5b0d659238807E84D0176930753d86), while access is authorized by [DAO Voting](https://etherscan.io/address/0x2e59A20f205bB85a89C53f1936454680651E618e) employing Aragon [ACL](https://etherscan.io/address/0x9895f0f17cc1d1891b6f18ee0b483b6f221b37bb) permission. And this permission controls only the source of the token transfer and doesn't cover any other constraints (i.e., limits, destinations, frequency of payouts), which could be considered too permissive for Easy Track voting model (requiring explicit objections to reject a vote, and otherwise let it pass).
+
+## Specification
+
+### Overview
+
+We propose to add permission for Easy Track on creating payments in the Finance contract that will limit the amount of funds that can be transferred from the treasury in one transaction using hard-coded constraints, implemented by logical permission parameters. 
```

Comment:
Looks like It's hard to read. I suggest splitting the sentence into two separate parts. 
Also, let's move 'implemented by logical permission parameters' to the third standalone sentence for simplicity.